zip-1014.rst

```diff
@@ -209,13 +209,16 @@ Zcash ecosystem, to perform major ongoing development (or other work) for the
 public good of the Zcash ecosystem, to the extent that such teams are available
 and effective.
 
-The funds SHALL be received and administered by ZF. ZF MUST disburse them as
-"Major Grants", but subject to the following additional constraints:
+The funds SHALL be received and administered by ZF. ZF MUST disburse them for
+"Major Grants" and expenses reasonably related to the administration of Major 
+Grants, but subject to the following additional constraints:
 
 1. These funds MUST only be used to issue Major Grants to external parties
-   that are independent of ZF. They MUST NOT be used by ZF for its internal
-   operations and direct expenses. Additionally, BP, ECC, and ZF are ineligible
-   to receive Major Grants.
+   that are independent of ZF, and to pay for expenses reasonably related to 
+   the administration of Major Grants. They MUST NOT be used by ZF for its 
+   internal operations and direct expenses not related to administration of 
+   Major Grants. Additionally, BP, ECC, and ZF are ineligible to receive 
```

Comment:
As with the added text on lines 213-214, this change is required in order to allow ZF to disburse funds from the Discretionary Budget.